crates/admin/src/cluster_controller/logs_controller.rs

```diff
@@ -302,6 +311,7 @@ fn build_new_replicated_loglet_configuration(
     loglet_id: ReplicatedLogletId,
     observed_cluster_state: &ObservedClusterState,
     previous_configuration: Option<&ReplicatedLogletParams>,
+    preferred_sequencer: Option<PlainNodeId>,
```

Comment:
If this is `Option<NodeId>`, the caller can be precise about the node generation or not.

Reply:
Good point. Will change it.